data/presets/cemetery/grave.json

```
@@ -0,0 +1,29 @@
+{
+    "icon": "maki-cemetery",
+    "fields": [
+        "name",
+        "inscription"
+    ],
+    "moreFields": [
+        "description"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "cemetery": "grave"
+    },
+    "terms": [
+        "burial ground",
+        "cemetary",
+        "cemetery",
+        "churchyard",
+        "columbarium",
+        "grave yard",
+        "graveyard",
+        "mausoleum",
+        "tomb"
```

I wouldn't just copy the terms of the Graveyard preset here.

Would it be better to only keep the terms which match an individual grave (and remove those terms from the `amenity=grave_yard` preset)?

```suggestion
        "mausoleum",
        "tomb"
```